packages/useful-types/README.md

```diff
@@ -60,7 +60,9 @@ The following type aliases are provided by this library:
   type SelectiveObj = Omit<Obj, 'foo' | 'bar'>; // {baz: number}
   ```
 
-- `Props<T>`: Extracts the prop type from a React component. This allows you to access property types without having to manually export/ import the type.
+- `Props<T>`: ⚠️ No longer availble.
```

Comment:
The list of things that aren't in this library is pretty long ;) I reckon this info is better placed in the changelog